config.template.yaml

```diff
@@ -34,17 +61,34 @@ CLIENT_URL: "http://127.0.0.1:8000/"
 # `AuthClientClassName` should be a class from `app.utils.auth.providers`
 # `secret` may be omitted to use PKCE instead of a client secret
 AUTH_CLIENTS:
+  Titan:
+    secret:
+    redirect_uri:
+      - http://localhost:3000/static.html
+      - http://127.0.0.1:3000/static.html
+    auth_client: AppAuthClient
+  Postman:
+    secret: PostmanSecret
+    redirect_uri:
+      - https://oauth.pstmn.io/v1/callback
+      - http://localhost:8000/docs/oauth2-redirect
+      - http://127.0.0.1:8000/docs/oauth2-redirect
+      - http://postman/
+    auth_client: APIToolAuthClient
 
 #####################
 # Hyperion settings #
 #####################
 
-LOG_DEBUG_MESSAGES: true
+LOG_DEBUG_MESSAGES: True
 
 # Origins for the CORS middleware. `["http://localhost"]` can be used for development.
 # See https://fastapi.tiangolo.com/tutorial/cors/
 # It should begin with 'http://' or 'https:// and should never end with a '/'
-CORS_ORIGINS: ["http://localhost"]
+CORS_ORIGINS:
+  - http://localhost:3000
+  - http://127.0.0.1:3000/
```

Comment:
We indicate a few lines earlier that these address should never end with a `/`. We should either change the requirement or the default value